test/functional/README.md

```diff
@@ -82,7 +82,7 @@ P2P messages. These can be found in the following source files:
 
 #### Using the P2P interface
 
-- `messages.py` contains all the definitions for objects that pass
+- [messages.py](test/functional/test_framework/messages.py) contains all the definitions for objects that pass
```

Comment:
Are the label and link inverted here?

Reply:
(the contents)

Reply:
No, but the link was a bit off. Fix coming shortly.